bridge/github/export_test.go

```diff
@@ -0,0 +1,375 @@
+package github
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/MichaelMure/git-bug/bridge/core"
+	"github.com/MichaelMure/git-bug/bug"
+	"github.com/MichaelMure/git-bug/cache"
+	"github.com/MichaelMure/git-bug/repository"
+	"github.com/MichaelMure/git-bug/util/interrupt"
+)
+
+const (
+	testRepoBaseName = "git-bug-test-github-exporter"
+)
+
+type testCase struct {
+	name    string
+	bug     *cache.BugCache
+	numOrOp int // number of original operations
+}
+
+func testCases(repo *cache.RepoCache, identity *cache.IdentityCache) ([]*testCase, error) {
+	// simple bug
+	simpleBug, _, err := repo.NewBug("simple bug", "new bug")
+	if err != nil {
```

Comment:
it's not wrong but if you pass the `t *testing.T` here you can do all the assert here and compact the code